### DISCUSSION TITLE
Who is responsible for which Python PIP package?

### DISCUSSION THREAD

**Discussion by ichorid** [arvidn/libtorrent#7178] [Q&A] [answered]
Hi, all! (especially @arvidn :wink: )
There is a mess of Libtorrent packages in Python PIP right now. Who is the author of which package there? Is there an official package?
I'm asking because I'd like to see Python 3.11 version of Libtorrent 2.0 and 1.x there, and I don't even know who to ask about it.

**Comment by arvidn** [comment]
I'm hoping @AllSeeingEyeTolledEweSew can help with that

**Comment by AllSeeingEyeTolledEweSew** [comment, marked as answer]
Hi. https://pypi.org/project/libtorrent/ is the official package. https://pypi.org/project/python-libtorrent/ was never official.

We're not building wheels for cpython 3.11 because boost-python is broken in 3.11. See https://github.com/arvidn/libtorrent/pull/7061#issue-1367122973.

boost-python fixed this in https://github.com/boostorg/python/pull/385, but for some reason the boost megaproject hasn't picked it up yet. I filed https://github.com/boostorg/boost/issues/706 about this.

**Reply by userdocs** [reply]
This should be fixed in 1.81.0 ?

https://github.com/boostorg/boost/releases/tag/boost-1.81.0

**Reply by ichorid** [reply]
https://github.com/arvidn/libtorrent/discussions/7187
